applications/luci-app-statistics/htdocs/luci-static/resources/view/statistics/plugins/ping.js

```diff
@@ -32,6 +32,12 @@ return baseclass.extend({
 		o.default = '30';
 		o.datatype = 'ufloat';
 		o.depends('enable', '1');
+
+		o=s.option(form.Value,'MaxMissed',_('Trigger a DNS resolve after the host has not replied to this number of packets'));
```

Comment:
I took this text from the manual page...if that's not OK I can rewrite it

Reply:
Looking at other pages, I found a better way to do this with more descriptive text, so I pushed a new change.
Looks like this now:
![image](https://user-images.githubusercontent.com/52104389/129464380-767ecac2-39e5-47e9-b811-f8f79248ec79.png)